game-app/game-core/src/main/java/games/strategy/triplea/attachments/RulesAttachment.java

```diff
@@ -1031,6 +1110,20 @@ private boolean checkTechs(final GamePlayer player, final TechnologyFrontier tec
     return found >= techCount;
   }
 
+  private boolean checkHaveResources(final List<GamePlayer> players) {
+    final boolean toSum =
+        haveResources[1].equalsIgnoreCase("sum") || haveResources[1].equalsIgnoreCase("add");
```

Comment:
This logic looks like it treat `sum` and `add` the same. Is that intentional?
If so, why? Is it for consistency with something else?

Reply:
Yes add and sum mean the same thing. Yes, it is intentional. They can be used interchangeably without forcing the map makers to chose.

Cheers...